app/controllers/contacts_controller.rb

```diff
@@ -20,7 +20,11 @@ def index
       format.json {
         @people = if params[:q].present?
                     mutual = params[:mutual].present? && params[:mutual]
-                    Person.search(params[:q], current_user, only_contacts: true, mutual: mutual).limit(15)
```

Comment:
I'm not sure that's the good place to put that changes. If the comment above is right, "Used for mentions" means an admin will be able to @username the user when writing a post. I am not sure that gives access to the post to the user, and it will have to be put in an aspect etc, that's not practical. I think private messages are what we want to achieve here.